File: w06/d04/Kyle_Sweet/hangman.js
function Hangman (word) {
  
  this.progress = function(){
    
    var playerProgress = [];
    for (var i = 0; i < word.length; i++) {
      playerProgress.push("_");
    }
    return playerProgress.join(" ");
};

  function guess(letter){
  if (guess === word) {
    return "true"
  }
  else if {
    return "false"
  };
  // }


  
var game = new Hangman("detective");
console.log(game.progress());
// console.log(game.guess("a"));
// console.log(game.guess("t"));

var game = new Hangman("gandhi");
console.log(game.progress());
  

File: w10/d01/Kyle_Sweet/hangman/lib/hangman.rb
class Hangman

@words = "detective", "gandhi"]

  def initialize(word)
    @word = @words.sample
    @progress = @word.map { "_" }
  end

  def progress()
    @progress.join(" ")
  end

  def guess?(guessed_letter)
    if @word.include?(guessed_letter)
      @word.each_with_index do |letter, index|
        if letter == guessed_letter
          @progress[index] = letter
        end
      end
      return true
    else
      return false
    end
  end

  def game_over
  end

end